harvard-university-for-the-creative-arts.csl

```diff
@@ -231,10 +236,15 @@
             <et-al font-style="italic"/>
           </names>
           <group delimiter=", ">
-            <group font-style="normal" font-variant="normal">
+            <group font-style="normal" font-variant="normal" delimiter=" ">
               <text variable="container-title" font-style="italic" suffix="."/>
               <text variable="collection-title" font-style="italic"/>
-              <text variable="volume" prefix=" Vol " suffix="."/>
+              <choose>
+                <if match="any" variable="volume">
+                  <text term="volume" form="short" text-case="capitalize-first"/>
+                  <text variable="volume" text-case="uppercase" prefix=" " suffix="."/>
+                </if>
+              </choose>
```

Comment:
@ibadger, because `<group/>` acts as an implicit conditional (see http://docs.citationstyles.org/en/stable/specification.html#group), you can simplify

```xml
              <choose>
                <if match="any" variable="volume">
                  <text term="volume" form="short" text-case="capitalize-first"/>
                  <text variable="volume" text-case="uppercase" prefix=" " suffix="."/>
                </if>
              </choose>
```

to

```xml
              <group>
                <text term="volume" form="short" text-case="capitalize-first"/>
                <text variable="volume" text-case="uppercase" prefix=" " suffix="."/>
              </group>
```

(the group won't output anything if all the variables it calls have no value)